docs/src/getting-started/dependencies.md

```diff
@@ -6,29 +6,18 @@
 
 To run the Fuel indexer, you'll need to install a few dependencies on your system:
 
-1. The [Rust programming language, with its package manager](https://www.rust-lang.org/tools/install)
-2. [`fuelup`](#fuelup), the Fuel toolchain manager
-3. A [PostgresQL](#postgresql) server backend
-4. The [`wasm32-unknown-unknown`](#web-assembly-wasm) `rustup` target
-5. [`wasm-snip`](#web-assembly-wasm), a utility for stripping symbols from WebAssemly binaries.
+1. The [Fuel toolchain](https://docs.fuel.network/guides/installation)
+2. A [PostgresQL](#postgresql) server backend
+3. The [`wasm32-unknown-unknown`](#web-assembly-wasm) `rustup` target
+4. [`wasm-snip`](#web-assembly-wasm), a utility for stripping symbols from WebAssemly binaries.
 
 > If you don't want to install a database directly onto your system, you can use Docker to run a database in an isolated container. You can install Docker by following its [installation instructions](https://docs.docker.com/get-docker/).
 >
 > For reference purposes, we provide a [`docker compose` file](https://github.com/FuelLabs/fuel-indexer/blob/develop/scripts/docker-compose.yaml) that comes with a PostgresSQL server and a Fuel indexer service.
 
-## `fuelup`
+## The `Fuel` toolchain
 
-We strongly recommend that you use the Fuel indexer that's made available via [`forc`, the Fuel orchestrator](https://fuellabs.github.io/sway/master/book/forc/index.html). You can get `forc` (and other Fuel components) by way of [`fuelup`, the Fuel toolchain manager](https://fuellabs.github.io/fuelup/latest).
-
-Install `fuelup` by running the following command, which downloads and runs the installation script.
-
-```bash
```

Comment:
@cold-briu 
- Any reason we removed this?
- If possible, it'd be preferable to keep the user on this page (no extra clicks) if installing `fuelup` is still possible in a single command.
  - Similarly for the "After `fuelup` has been installed," bit below. Are we throwing the baby 👶🏼  out with the bath water 🛀🏼   here?

Reply:
To extend @ra0x3's point, the intention was to (when possible) prevent the user from having to go to another page just to come back to this one after they install fuelup. I think there's a middle ground where we include the link but also include the one-line installation command; that way, they can open it if they want more information.

Reply:
Hi there! Thanks for reviewing. 

- The main reason for this is to reduce content duplication across the entire toolchain documentation, simplifying maintenance.

- I understand your perspective on keeping the user in the same page. 

- My take is to use mdbook code imports as seen [here](https://github.com/FuelLabs/sway/blob/master/docs/book/src/advanced/assembly.md).

Reply:
Hi there! 

In the short term, we won't take further action here since it is common that the user already have the toolchain installed by the moment they start working with the indexer.

I'll request review again, so it can be merged.